File: Number of 1 Bits/cpp/code.cpp
class Solution {
public:
    int hammingWeight(int n) {
        int size = 32;
        int counter = 0;

        int i = 0;
        while (i <= 32) {
            if (n & 1)
                counter++;
            n = n >> 1;
            i++;
        }
        return counter;
    }
};

File: Number of 1 Bits/javaScript/code.js
/**
 * @param {number} n
 * @return {number}
 */
var hammingWeight = function(n) {
    let counter = 0;
    let size = 32;

    for (let i = 0; i < size; i++) {
        if ((n & 1) === 1) {
            counter++;
        }
        n = n >> 1;
    }

return counter;    
};
